airbyte-integrations/connectors/source-file/source_file/client.py

```diff
@@ -396,6 +397,12 @@ def read(self, fields: Iterable = None) -> Iterable[dict]:
             except ConnectionResetError:
                 logger.info(f"Catched `connection reset error - 104`, stream: {self.stream_name} ({self.reader.full_url})")
                 raise ConnectionResetError
+            except ProtocolError as err:
```

Comment:
Do we have the unit_test for this  case?

Reply:
unit test added